cmake/configure.cmake

```diff
@@ -111,9 +148,29 @@ if(UNIX)
     list(APPEND MK_LIBS resolv)
   endif()
 
-  list(APPEND MK_LIBS GeoIP crypto ssl event event_openssl event_pthreads)
+  # The _pthreads library is only built on Unix. The _openssl library does not
+  # seem to be generated by the cmake script used on Windows.
+  list(APPEND MK_LIBS event_pthreads)
+  list(APPEND MK_LIBS event_openssl)
+  list(APPEND MK_LIBS ssl)
+  list(APPEND MK_LIBS crypto)
+  list(APPEND MK_LIBS event_core)
+  list(APPEND MK_LIBS event_extra)
+
+elseif(WIN32)
+  # On Windows we expect a specific version of libressl. We can possibly make
```

Comment:
I don't see anything in this PR that provides guidance on how to get this version of these libraries.  Is this just 'gonna work' in that cmake will get the right version? or do i need to have these libs / where can i get them?

build/spec/libressl  has a URL for `libressl-2.5.5.tar.gz` which is not obviously the same

Reply:
Good question! So, that ABI number is actually equal to v2.5.5. I have discovered anyway that it seems it's not needed to clearly specify the ABI, so it would probably be better to drop the explicit ABI number, to make the thing overall more maintainable.